src/space-picker/space-picker.js

```diff
@@ -169,6 +169,17 @@ const Self = class SpacePicker extends ColorElement {
 				return ret;
 			},
 		},
+
+		getSpaceLabel: {
+			type: {
+				is: Function,
+				arguments: ["space"],
+			},
+			default () {
+				return space => space.name;
```

Comment:
Also, how does Nude Element handle default values with the function type? (since usually default values mean getter)

Reply:
> Also, how does Nude Element handle default values with the function type? (since usually default values mean getter)

If I get the question correctly, in that case, everything is correctly resolved:

https://github.com/nudeui/element/blob/3c3297eed2506200998b43e29eaa0512c1f0bdcc/src/props/Prop.js#L166

and parsed

https://github.com/nudeui/element/blob/3c3297eed2506200998b43e29eaa0512c1f0bdcc/src/props/types/basic.js#L47-L50

So when we call props with such values in the code, they are already handled correctly.